packages/main/test/pages/ColorPicker.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
Can you please add
- sample with listening to the color change event
- Playground sample
- Few tests